### PR TITLE
PLAT-1271 | Run progress:store once a day

### PR DESCRIPTION
**Pull request by rogatty**
https://bethink.atlassian.net/browse/PLAT-1271
https://github.com/bethinkpl/ansible/pull/68
https://github.com/bethinkpl/ansible/pull/69

### CODE REVIEW

File: resources/assets/js/components/user/UserStats.vue [hunk 1]
```diff
@@ -22,15 +22,17 @@
...
-		<div>Przerobionych lekcji: {{lessonsCompletedPerc}}% ({{lessonsCompleted}}/{{lessonsTotal}})</div>
-		<div>Rozpoczętych lekcji: {{lessonsStartedPerc}}% ({{lessonsStarted}}/{{lessonsTotal}})</div>
-		<div class="strong margin top">Baza Pytań</div>
-		<div>Rozwiązanych pytań: {{questionsSolvedPerc}}% ({{questionsSolved}}/{{questionsTotal}})</div>
-		<div class="strong margin top">Społeczność</div>
-		<div>Liczba wątków: {{totalSocial}}</div>
+		<p class="strong">Ogólne</p>
+		<p>Liczba spędzonych minut: {{timeSpent}}</p>
+		<p class="strong margin top">Lekcje</p>
+		<p>Dane aktualizują się raz dziennie, około 4:00 rano.</p>
```
Comment (kvas-damian) [inline resources/assets/js/components/user/UserStats.vue]: what about daylight saving time?

Comment (rogatty) [reply resources/assets/js/components/user/UserStats.vue]: 😱

File: app/Console/Kernel.php [hunk 2]
```diff
@@ -64,15 +64,14 @@ protected function schedule(Schedule $schedule)
 
 		$schedule
 			->command('progress:store')
-			->hourly()
-			->withoutOverlapping()
+			->dailyAt('02:00')
```
Comment (rogatty) [inline app/Console/Kernel.php]:
```
igor@worker-1:~$ date
Thu May  2 08:37:41 UTC 2019
```

File: resources/assets/js/components/user/UserStats.vue (hunk 1 above)
Comment (kvas-damian) [reply resources/assets/js/components/user/UserStats.vue]: Isn't it enough to say "w nocy"?

Comment (rogatty) [reply resources/assets/js/components/user/UserStats.vue]: Fixed in https://github.com/bethinkpl/wnl-platform/pull/1462/commits/8f106e80cfdb875333e9cb6ff26f3048af7098a9

Comment (bognix) [reply resources/assets/js/components/user/UserStats.vue]: If we have some communication related with this message we can update the code. Let's leave it as it is for now.

**Pull request merged**